internal/nodes/avi_model_nodes_utils.go

```diff
@@ -151,6 +202,30 @@ func (o *AviVsNodeCommonFields) ConvertToRef() {
 	}
 }
 
+func (o *AviVsNodeGeneratedFields) ConvertL7RuleFieldsToNil() {
+	if o != nil {
+		o.AllowInvalidClientCert = nil
+		o.BotPolicyRef =nil
+		o.CloseClientConnOnConfigUpdate =nil
```

Comment:
Is it ok to set boolean fields to nil as well ?

Reply:
yes. In UT we test them for nil value as well.

Reply:
Problem wont be there in Uts, but might be seen during FTs, once you do end to end testing. 
Specifically during object creation at controller's end. I have faced issues in tha past when everything looked okay at AKO's end but still objects didnt get created at controller.
We can close this comment once you have tested CRUD end to end with these values.